SNIClient.py

```diff
@@ -342,17 +360,16 @@ async def _snes_connect(ctx: SNIContext, address: str, retry: bool = True) -> We
             return snes_socket
         if not retry:
             break
+    return None
 
 
 class SNESRequest(typing.TypedDict):
     Opcode: str
     Space: str
-    Operands: typing.List[str]
-    # TODO: When Python 3.11 is the lowest version supported, `Operands` can use `typing.NotRequired` (pep-0655)
-    # Then the `Operands` key doesn't need to be given for opcodes that don't use it.
+    Operands: list[str]
```

Comment:
I thought about doing what was in this `TODO` comment (that I wrote a long time ago).
But then I figured it would be safer, and easier, and more consistent to just include the `Operands` key even if it's not necessary.

The only downside is sending a few extra bytes to SNI, only when we make the initial connection.